File: src/main/kotlin/io/xhub/confIOMessenger/domain/QuickReply.java
package io.xhub.confIOMessenger.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class QuickReply {
    @JsonProperty("content_type")
    private String contenType;
    private String title;
    @JsonProperty("image_url")
    private String imageUrl;
    private String payload;

    public String getContenType() {
        return contenType;
    }

    public void setContenType(String contenType) {
        this.contenType = contenType;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getPayload() {
        return payload;
    }

    public void setPayload(String payload) {
        this.payload = payload;
    }
}

File: src/main/kotlin/io/xhub/confIOMessenger/domain/QuickReply.kt
package io.xhub.confIOMessenger.domain

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.core.util.RequestPayload
import io.xhub.confIOMessenger.annotation.IgnoreNotNullUnknown

@IgnoreNotNullUnknown
data class  QuickReply constructor(
        @JsonProperty("payload")
        val payload: String?,
        @JsonProperty("title")
        val title:String?=null,
        @JsonProperty("content_type")
        val content_type:String?="text",
        @JsonProperty("image_url")
        val image_url:String?=null

)